File: kattis/inputoutput/hello/hello.py
class hello:

    def __init__(self, debug: bool = False):
        self.debug = debug

    def dprint(self, *args, **kwargs):
        if self.debug:
            print(*args, **kwargs)

    def generateOutput(self):
        # Print Hello World!
        self.dprint("-------------------------------------")
        print("Hello World!")
        self.dprint("-------------------------------------")

if __name__ == "__main__":
    hello(False).generateOutput()

File: kattis/inputoutput/hello/hello.java
package kattis.inputoutput.hello;

public class hello {

    private final boolean debug;

    public hello(boolean debug) {
        this.debug = debug;
    }

    private void dprint(Object... args) {
        if (debug) {
            StringBuilder sb = new StringBuilder();
            for (Object arg : args) {
                sb.append(arg).append(" ");
            }
            System.out.println(sb.toString().trim());
        }
    }

    public void generateOutput() {
        // Print Hello World!
        dprint("-------------------------------------");
        System.out.println("Hello World!");
        dprint("-------------------------------------");
    }

    public static void main(String[] args) {
        new hello(false).generateOutput();
    }
}